### ISSUE TITLE
Launch after update launches ONLY if there is an update.

### ISSUE THREAD

**Issue by Xarishark** [amakvana/EzYuzu#25]
It should check for an update and if there is no update the launch instantly! The whole point is to run EzYuzu every time you run yuzu to check for updates or **just launch instantly if there are no updates!**

**Comment by amakvana** [comment]
Thanks for the feedback! 

It does do this via command line, if you run the following switches 
```
EzYuzu.exe -p "\\path\to\yuzu\folder" -l
```
However, I shall tweak the GUI side to do exactly this

**Comment by Xarishark** [comment]
Does that work like a -silent flag? will it show the interface if there are no updates? if not you could implement a -silent flag that wont even show the interface if there are no update and just run the emulator instantly. 
That way you only see the EzYuzu UI when there are updates. Cleaner that way. Of course, you can still straight run the interface without the flag to make it work the same way it works now.

**Comment by Xarishark** [comment]
Updated the title to make it clear that I mean a UI feature request.

**Comment by amakvana** [comment]
> I opened the exe and directed it to yuzufolder. Then clicked the launch after update and closed / relaunched it to test it. It threw an error ![image](https://user-images.githubusercontent.com/21146743/240637575-de176f86-a61e-498a-9c22-339270dabf51.png)

I can't reproduce the error on my end using the same executable attached. I've posted a gif replicating the steps you did, but it loads on my end

![ezyuzu-troubleshoot](https://github.com/amakvana/EzYuzu/assets/66264136/8238f041-c0a8-4d39-a2f9-21bcb62fa7ad)

**Comment by Xarishark** [comment]
Yeah Im using the testbuild you send me. Im doing the same steps yes. I had to run it as admin for it to work but I dont have to do the same thing with the mainline version.

**Comment by Xarishark** [comment]
welp now the mainline shortcut stopped working too.....

**Comment by amakvana** [comment]
Try rebooting your machine and check again because I'm struggling to replicate the issue on my end

**Comment by Xarishark** [comment]
i thinking something is very slow on asnwering and its timming out. Can you send me the links it tries to resolve so I ping them and check if something is wrong on my end connection wise ?

**Comment by amakvana** [comment]
> i thinking something is very slow on asnwering and its timming out. Can you send me the links it tries to resolve so I ping them and check if something is wrong on my end connection wise ?

Sure, these are the URL's EzYuzu connects to 

<b>App Version Checking (checks if EzYuzu is latest version):</b>
https://raw.githubusercontent.com/amakvana/EzYuzu/master/version

<b>Json Data:</b>
https://api.github.com/repos/pineappleEA/pineapple-src/releases
https://api.github.com/repos/yuzu-emu/yuzu-mainline/releases

<b>Dependencies:</b>
https://raw.githubusercontent.com/amakvana/EzYuzu/master/assets/7z/22.01/7z.zip
https://aka.ms/vs/16/release/vc_redist.x64.exe

**Comment by amakvana** [comment]
I've added debug prompts all over and also got it set up to write logs with the name of the function triggered from

Definitely sounds like something on your end. Have you got any proxy settings configured? 

[EzYuzu_Xarishark.zip](https://github.com/amakvana/EzYuzu/files/11556475/EzYuzu_Xarishark.zip)

**Comment by Xarishark** [comment]
Thats weird that exe works fine. I dont get crashes anymore. Be sure to kill the EzYuzu executable when you run yuzu.

**Comment by amakvana** [comment]
Closed - Fixed in v1.6.1.0
https://github.com/amakvana/EzYuzu/releases/latest
